powertools-sqs/pom.xml

```diff
@@ -15,6 +15,7 @@
 
     <name>Powertools for AWS Lambda (Java) library SQS</name>
     <description>
+        Deprecated: Batch processing is now handled in powertools-batch and large messages in powertools-large-messages modules.
```

Comment:
We're going to have to merge batch before we do our next release too :)

Reply:
sure, next release will contain batch and large message "v2ish"